spec/purpose_and_scope.md

```diff
@@ -362,11 +363,10 @@ standard can become final. We prefer to delay this decision, to see how
 NEP 37 adoption will work out.
 ```
 
-The `xp` namespace must contain the array object and all functionality
```

Comment:
I removed "array object" here, because the array class is not itself required to be a name in the namespace.